src/Dropzone/tests/Form/DropzoneTypeTest.php

```diff
@@ -38,21 +38,14 @@ public function testRenderForm()
         $rendered = $container->get(Environment::class)->render('dropzone_form.html.twig', ['form' => $form->createView()]);
 
         $this->assertSame(
-            '<form name="form" method="post" enctype="multipart/form-data"><div id="form"><div><label for="form_photo" class="required">Photo</label><div class="dropzone-container" data-controller="mydropzone symfony--ux-dropzone--dropzone">
-        <input type="file" id="form_photo" name="form[photo]" required="required" data-controller="" class="dropzone-input" data-symfony--ux-dropzone--dropzone-target="input" />
-
-        <div class="dropzone-placeholder" data-symfony--ux-dropzone--dropzone-target="placeholder"></div>
-
-        <div class="dropzone-preview" data-symfony--ux-dropzone--dropzone-target="preview" style="display: none">
-            <button class="dropzone-preview-button" type="button"
-                    data-symfony--ux-dropzone--dropzone-target="previewClearButton"></button>
-
-            <div class="dropzone-preview-image" style="display: none"
-                 data-symfony--ux-dropzone--dropzone-target="previewImage"></div>
-
-            <div data-symfony--ux-dropzone--dropzone-target="previewFilename" class="dropzone-preview-filename"></div>
-        </div>
-    </div></div></div></form>
+'<form name="form" method="post" enctype="multipart/form-data"><div id="form"><div><label for="form_photo" class="required">Photo</label><div class="dropzone-container" data-controller="mydropzone symfony--ux-dropzone--dropzone">
```

Comment:
This is the proof we cannot do these changes without a dedicated form theme and/or controller js : the structural changes in the HTML are going to break layout/tests of people currently using this component

Reply:
💯

Changing the DOM and/or styles like in this way is a no go.